File: codewars/8kyu/squash-the-bugs/dduck-squash-the-bugs.py
# https://www.codewars.com/kata/56f173a35b91399a05000cb7/

from unittest import TestCase

find_longest = lambda string: len(sorted(string.split(), key=len).pop())

TestCase().assertEqual(find_longest("The quick white fox jumped around the massive dog"), 7)
TestCase().assertEqual(find_longest("Take me to tinseltown with you"), 10)
TestCase().assertEqual(find_longest("Sausage chops"), 7)
TestCase().assertEqual(find_longest("Wind your body and wiggle your belly"), 6)
TestCase().assertEqual(find_longest("Lets all go on holiday"), 7)

File: codewars/8kyu/squash-the-bugs/dduck-squash-the-bugs.js
// https://www.codewars.com/kata/56f173a35b91399a05000cb7/train/javascript
const findLongest = (words) => words.split(' ').sort((a, b) => a.length - b.length).pop().length;

console.log(findLongest("The quick white fox jumped around the massive dog"), 7);
console.log(findLongest("Take me to tinseltown with you"), 10);
console.log(findLongest("Sausage chops"), 7);
console.log(findLongest("Wind your body and wiggle your belly"), 6);
console.log(findLongest("Lets all go on holiday"), 7);
